src/client.js

```diff
@@ -494,6 +493,8 @@ Client.prototype.clientAttach = function(socketId) {
 		});
 	}
 
+	client.attachedClients[socketId] = client.lastActiveChannel;
+
```

Comment:
> 18:13	**astorije**	Just curious, why did you move this?
> 18:13	**xPaw**	look at line 487
> 18:14	**xPaw**	it added to array, then compared to see if size of array is 0, which it would be 1
> 18:14	**xPaw**	I could change the comparison to 1, or move assignment